google_drive/tests/integration/xml/image.xml

```diff
@@ -0,0 +1,3 @@
+<vertical_demo>
+    <google-document embed_code="&lt;img src='https://docs.google.com/drawings/d/1lmmxboBM5c_0WCTjhAxBdkpqQb3T8VSwtuG0TRR1ODQ/pub?w=960&amp;h=720'&gt;"/>
```

Comment:
I don't think so - this is the image to use for testing. So, definitely not settings file. I like constant or property.